scripts/services/getConfig.ts

```diff
@@ -188,10 +269,10 @@ export const getConfig: T.ExpectedExports.getConfig = compat.getConfig({
             nullable: true,
             name: "Max Peers Incoming",
             description:
-              "Maximum number of simultaneous peers connecting inbound to the Monero daemon <br/><b>Default:</b> 16 (Monero's default is unlimited but we prefer a ceiling to limit network surveillance from spy nodes and to discourage excessive bandwidth consumption)",
+              "Maximum number of simultaneous peers connecting inbound to the Monero daemon <br/><b>Default:</b> 18 (Monero's default is unlimited but we prefer a ceiling to limit network surveillance from spy nodes and to discourage excessive bandwidth consumption)",
```

Comment:
```suggestion
              "Maximum number of simultaneous peers connecting inbound to the Monero daemon <br/><b>Default:</b> 24 (Monero's default is unlimited but we prefer a ceiling to limit network surveillance from spy nodes and to discourage excessive bandwidth consumption)",
```

double inc is min recommended

Reply:
Interesting. By whom?  Or where can I find this recommendation?

Reply:
Example
``` --out-peers 32 --in-peers 64 

If your network connection's upload bandwidth is less than 10 Mbit, use --out-peers 8 --in-peers 16 instead.

```
from p2pool.io/help